lib/matplotlib/tests/test_axes.py

```diff
@@ -743,7 +743,7 @@ def test_pcolormesh_datetime_axis():
     fig.subplots_adjust(hspace=0.4, top=0.98, bottom=.15)
     base = datetime.datetime(2013, 1, 1)
     x = np.array([base + datetime.timedelta(days=d) for d in range(21)])
-    y = np.arange(21)
+    y = range(21)
```

Comment:
look at lines (753 - 754) and (779 - 780).  This is blown up into a 2D thing before we pass it in to test.